.buildkite/nightly_steps.yml

```diff
@@ -96,3 +96,9 @@ steps:
       - ".buildkite/run_nigthly.sh dropbox"
     artifact_paths:
       - "perf8-report-*/**/*"
+  - label: "🔨 OneDrive"
+    command:
+      - ".buildkite/run_nigthly.sh onedrive"
+    artifact_paths:
+      - "perf8-report-*/**/*"
```

Comment:
Need to also add it to `.buildkite/pipeline.yml` following the pattern that connectors follow there

Reply:
Thanks, Done.